src/common/k8s-api/json-api.ts

```diff
@@ -194,11 +205,12 @@ export class JsonApi<D = JsonApiData, P extends JsonApiParams = JsonApiParams> {
   protected writeLog(log: JsonApiLog) {
     if (!this.config.debug) return;
     const { method, reqUrl, ...params } = log;
-    let textStyle = "font-weight: bold;";
+    // let textStyle = "font-weight: bold;";
```

Comment:
Do we want to leave these commented lines?

Reply:
No, I left these on purpose to get feedback... see https://github.com/lensapp/lens/pull/3553#discussion_r681474267